rca/wagtailapi/endpoints.py

```diff
@@ -18,5 +17,5 @@ class PagesAPIEndpoint(endpoints.PagesAPIEndpoint):
         FieldsFilter,
         RestrictedChildOfFilter,
         RestrictedDescendantOfFilter,
-        SearchFilter,
+        filters.SearchFilter,
```

Comment:
Are there other pages that rely on API changes that we may be potentially breaking?

Reply:
Nope

Reply:
At this moment no - we will be exposing pretty much all pages we create in the future though